### ISSUE TITLE
Spring AI projects page mispelled Anthropic "Claud" (should be "Claude")

### ISSUE THREAD

**Issue by christopher-nash** [spring-projects/spring-ai#1289]
I'm not sure where else to report this, so please move if not appropriate here.

At https://spring.io/projects/spring-ai it has a typo for Anthropic "Claud", which is incorrect. It should be Anthropic "Claude" (with and "e" at the end).

See https://www.anthropic.com/claude

![image](https://github.com/user-attachments/assets/9029ec1d-3c15-40e9-bb88-008c6068203d)

**Comment by inshining** [comment]
@markpollack 

I'm new to open source contributing. 
I'm trying to resolve this issue. 
But, the documentation at https://spring.io/projects/spring-ai doesn't seem to be maintained by this repo. 
Can you give me any hints?

**Comment by christopher-nash** [comment]
Yeah - that was the struggle I had as well in not knowing where the those docs are versioned.

**Comment by AgusCastro** [comment]
Hi all,
Checking similar issues(#1046), I found that the project's description docs are internal for the Spring team only.

@tzolov, hello! You solved a similar issue in the past. Do you know where this typo can be reported?

**Comment by dafriz** [comment]
Just noting that the typo has been fixed so this issue can be closed.

**Comment by String369** [comment]
Can I contribute to this issue.

**Comment by ilayaperumalg** [comment]
This page has been re-designed and modified. This issue is no longer relevant. Closing now.
